src/AmplifyGeofenceControl/index.ts

```diff
@@ -35,8 +35,8 @@ export class AmplifyGeofenceControl {
   _geofenceCreateInput?: HTMLElement;
   _addGeofenceContainer?: HTMLElement;
 
-  constructor(options: AmplifyGeofenceControlOptions) {
-    this._geofenceCollectionId = options.geofenceCollectionId ?? "fixme"; // this should be retrieved from Geofence API
+  constructor(options?: AmplifyGeofenceControlOptions) {
+    this._geofenceCollectionId = options?.geofenceCollectionId ?? "fixme"; // this should be retrieved from Geofence API
```

Comment:
What is happening here with the `fixme` and comment?

Reply:
Whoops this can actually be removed :)